### ISSUE TITLE
Confirmation email to corresponding author possibly sent multiple times

### ISSUE THREAD

**Issue by giorgiosironi** [elifesciences/elife-xpub#1207]
**Describe the bug**
Every (auto) saving of the manuscript seems to send a new email to the address entered in the first step of the initial submission.

**Steps To Reproduce**
1. Start an initial submission
2. Enter first name, last name, email address and institution
3. Wait a few seconds
4. Read the log line `info: Sent confirmation email to corresponding author: g.sironi@elifesciences.org` written multiple times

**Expected behavior**
If https://github.com/elifesciences/elife-xpub/pull/1206 is correct this means an email is going out on each autosave of the form.

**Screenshots**
Some more logs I have inserted:
```
info: manuscript.getAuthor(): null
info: newAuthorEmails: g.sironi@elife
info: originalAuthorEmails: 
debug: Updated manuscript manuscriptId=7270b619-fd16-4fae-8771-18fd64efdcb6, userId=75c11a2d-9185-4b6a-a68d-86a7f60eae64
info: POST /graphql 200 UpdateManuscript 1410 218.026 ms

info: Sent confirmation email to corresponding author: g.sironi@elife
info: manuscript.getAuthor(): [object Object]
info: newAuthorEmails: g.sironi@elifesciences.org
info: originalAuthorEmails: [object Object]
debug: Updated manuscript manuscriptId=7270b619-fd16-4fae-8771-18fd64efdcb6, userId=75c11a2d-9185-4b6a-a68d-86a7f60eae64
info: POST /graphql 200 UpdateManuscript 1428 149.295 ms

info: Sent confirmation email to corresponding author: g.sironi@elifesciences.org
```
It appears that there is a comparison between `[object Object]` and actual strings around https://github.com/elifesciences/elife-xpub/blob/develop/server/xpub-server/entities/manuscript/resolvers/updateManuscript.js#L24

Moreover, the autosave (if there is one) may be firing too soon saving partial results like `g.sironi@elife` rather than `g.sironi@elifesciences.org`.

**Comment by giorgiosironi** [comment]
The email infrastructure has been fixed in #890, so now actual multiple emails continue to go out:
![8a6a39e3-f979-4188-9dce-ba9727f82737](https://user-images.githubusercontent.com/160299/50081629-df2ec180-01e6-11e9-860c-51b6eed9fa2c.png)

**Comment by diversemix** [comment]
@giorgiosironi Thanks for the detailed report - @hdrury1 - can we move this to the top of the list
